iot-hub/Samples/device/PnpDeviceSamples/TemperatureController/Parameter.cs

```diff
@@ -14,7 +14,7 @@ namespace Microsoft.Azure.Devices.Client.Samples
     internal class Parameters
     {
         [Option(
-            's',
+            'f',
```

Comment:
'f' for Flow?

Reply:
Btw, if we can't find a good single letter because of conflicts, we don't have to have one. I'd rather not have one than one that doesn't make obvious sense.